designsafe/apps/rapid/openTopo_Data/scripts/centerpoint_datagenerator.py

```diff
@@ -0,0 +1,154 @@
+import requests
```

Comment:
can a README.md be added? 

It could include some install and run instructions, something like:
```
# Something

This script does something..

# Run

To install and run in a virtual environment: 
  ```python -m venv venv
        source venv/bin/activate
        pip install -r centerpoint_Requirements.txt
        python centerpoint_datagenerator.py
    ```
```

this can be helpful for us in the future (and for PR reviewers).